argus/recipes/cloud/windows.py

```
@@ -41,6 +41,22 @@ def wait_for_boot_completion(self):
         LOG.info("Waiting for first boot completion...")
         self._backend.remote_client.manager.wait_boot_completion()
 
+    def set_mtu(self, interface="ipv4",
+                subinterface_name="Ethernet",
+                mtu_value=1400,
+                store_type='active'):
+        set_mtu_cmd = ('netsh interface {interface_type} set subinterface '
+                       '"{name}" mtu={value} '
+                       'store={type}'.format(interface_type=interface,
+                                             name=subinterface_name,
+                                             value=mtu_value,
+                                             type=store_type))
+        LOG.info("Setting the MTU.")
+        try:
+            self._backend.remote_client.run_command_with_retry(set_mtu_cmd)
+        except Exception:
+            LOG.debug('MTU failed to set.')
```

A very bad practice, you hide the real exception message.